packages/yourloops/components/switch-role/switch-role-consequences-dialog.tsx

```diff
@@ -59,15 +72,24 @@ function SwitchRoleConsequencesDialog(props: SwitchRoleDialogProps): JSX.Element
   const handleAccept = () => onResult(true);
 
   return (
-    <Dialog id="switch-role-consequences-dialog" open={open} onClose={handleClose}>
+    <Dialog
+      id="switch-role-consequences-dialog"
+      className={classes.dialog}
+      PaperProps={{
+        style: {
+          padding: "20px",
+        },
+      }}
+      maxWidth="md"
+      open={open}
+      onClose={handleClose}>
       <DialogTitle id="switch-role-consequences-dialog-title">
-        <strong>{t("modal-switch-hcp-title")}</strong>
+        <strong>{t(title)}</strong>
```

Comment:
I would have put the translation call in the caller component.
It makes easier to find translations strings & fix translation problems.

Reply:
i think it makes more sense to keep it in the component. Otherwise it will be a mixed.